### ISSUE TITLE
Cannot select language from FO after importing a localization pack

### ISSUE THREAD

**Issue by nesrineabdmouleh** [PrestaShop/PrestaShop#19893]
#### Describe the bug

Cannot select a language from FO after importing a localization pack.

#### Expected behavior

When clicking on the language list the customer can select another language.

#### Steps to Reproduce

1. Go to 'International>Localization' page
2. Import a localization pack( Chile for expl)
3. Go to FO
4. Change the FO language -> KO cannot click on the list

**Screenshots**

https://drive.google.com/file/d/1OnDfK9dULcA6xbijumJn7vUF44qhwb2a/view

#### Additional information

* PrestaShop version: 1.7.7.x
* PHP version: 7.2

**Comment by Progi1984** [comment]
Any JS error ?

**Comment by matks** [comment]
```
jquery.live-polyfill.min.js:6 Uncaught ReferenceError: migrateWarn is not defined
    at new e.fn.init (jquery.live-polyfill.min.js:6)
    at e (jquery.js:152)
    at Function.e._getParentFromElement (theme.js:1)
    at Function.e._clearMenus (theme.js:1)
    at HTMLButtonElement.e.toggle (theme.js:1)
    at HTMLDocument.dispatch (jquery.js:5237)
    at HTMLDocument.m.handle (jquery.js:5044)
```

**Comment by matks** [comment]
May be fixed by https://github.com/PrestaShop/PrestaShop/pull/19896

**Comment by Progi1984** [comment]
Fixed by #19896
